v0/destinations/iterable/transform.js

```diff
@@ -513,12 +477,21 @@ const processRouterDest = async inputs => {
 
   // batching identifyArrayChunks
   let identifyBatchedResponseList = [];
-  if (identifyArrayChunks.length) {
+  if (identifyEventChunks.length) {
+    // arrayChunks = [[e1,e2,e3,..batchSize],[e1,e2,e3,..batchSize]..]
+    // transformed payload of (n) batch size
+    const identifyArrayChunks = _.chunk(
+      identifyEventChunks,
+      IDENTIFY_MAX_BATCH_SIZE
+    );
     identifyBatchedResponseList = await batchEvents(identifyArrayChunks);
```

Comment:
```suggestion
    identifyBatchedResponseList = batchEvents(identifyArrayChunks);
```
`batchEvents` is not an async function so await is not required.

Reply:
Yeah, updated